packages/core/src/VNodes/RangeNode.ts

```diff
@@ -0,0 +1,10 @@
+import { VNode, VNodeType } from './VNode';
+
+export class RangeNode extends VNode {
+    properties = {
+        atomic: true,
+    };
+    constructor(tailOrHead: 'tail' | 'head') {
```

Comment:
Why use string rather than an ENUM?

Reply:
Again, removed later. What happened with this review?